DB/src/main/java/io/deephaven/db/v2/select/AbstractConditionFilter.java

```diff
@@ -177,6 +200,7 @@ public Index filter(Index selection, Index fullSet, Table table, boolean usePrev
     }
 
     protected abstract Filter getFilter(Table table, Index fullSet) throws InstantiationException, IllegalAccessException, NoSuchMethodException, InvocationTargetException;
+    protected abstract void setFilter(Filter filter);
```

Comment:
I think we'll need to get somebody more familiar w/ engine stuff to weigh in if this is a pattern we want / should even use. I'm don't grok the scope of this change wrt the numba stuff. @rcaudy may need to spend some time reviewing to understand impact.